docs/packaging-a-RNComponent-ios.md

```diff
@@ -0,0 +1,65 @@
+---
+id: packaging-a-RNComponent-ios
+title: Packaging a react-native component
+layout: docs
+category: Guides (iOS)
+permalink: docs/packaging-a-RNComponent-ios.html
+next: native-modules-android
+---
+
+You can also inject Native modules into react-native by creating a project in Objective-C and publish it as package in npm so that anyone can use it.
+
+*Please proceed with the following steps.*
+### Creating the package
+
+* Create a new project, use this type of naming, React-native(RN) prefix.
+
+![new-project](http://brentvatne.ca/images/packaging/1-new-project.png)
+![new-project-2](http://brentvatne.ca/images/packaging/2-project-name.png)
```

Comment:
I'd avoid images at all costs for future maintainability. If you can say it with simple words, use that.